src/charts/chartOptions.js

```diff
@@ -367,6 +366,16 @@ define(['jquery', 'common/rivetsExtra', "charts/chartWindow", "charts/charts", '
                     });
                 }
 
+                // Add event only once.
+                !isListenerAdded && $('body').on('click', function(event){
```

Comment:
I tried that but it didnt work because of event propagation. The propagating event caused the _one_ event to execute right after it was added.